config/spryk/spryks/Spryker/AddModuleComposerJson.yml

```diff
@@ -1,6 +1,6 @@
 spryk: template
 description: "Adds a new composer.json to the Modules root."
-mode: core
+mode: "core|sdk|eco"
```

Comment:
I was under the assumption, the mode flag is **bool** in terms of Core vs everything else.
In core it still can have hundreds of **Namespace** strings, but those should be separated from this.
Otherwise you create a huge conflict everytime we introduce a new core namespace.
Please keep those separate.

Reply:
Your assumption is wrong, apparently. I also thought it is more open but it was and is still hardcoded.

We can update this in a next step and resolve the paths by organization or other user input.

Reply:
This part definitely needs more flexibility.

Reply:
:/ Thats really bad
This was not supposed to be like this, ever.

Can we do that right now, before making this even worse? Now "mode" is a list which it never should have been...
Introduce the org namespace and use the combination of bool mode + namespace to determine correct vendor folder?

Reply:
I can create a follow up for this. IMO mode shouldn't be present at all.

Reply:
Why not? This is the correct bool to switch project vs non project specific things.
All the rest is just the right folder name as per namespace, as I just pointed out.

I think this should be fixed right now. The SDK support is not that important to go off into the wrong direction instead of just fixing up this by keeping mode as it is, and enabling the namespace into it as addition to find the right vendor path.